index.js

```diff
@@ -84,8 +84,8 @@ module.exports = {
     includer.options.includeFileExtensionInSnippetNames = includer.options.includeFileExtensionInSnippetNames || false;
     includer.options.snippetSearchPaths = includer.options.snippetSearchPaths || ['tests/dummy/app'];
     includer.options.snippetRegexes = Object.assign({}, {
-      begin: /{{#(?:docs-snippet|demo.example)\sname=(?:"|')(\S+)(?:"|')/,
-      end: /{{\/(?:docs-snippet|demo.example)}}/,
+      begin: /(?:{{#|<)(?:docs-snippet|demo.example)\s@?name=(?:"|')(\S+)(?:"|')/,
+      end: /(?:{{|<)\/(?:docs-snippet|demo.example)(?:}}|>)/,
```

Comment:
For this to work for standalone snippets, we'd need to accept `DocsSnippet` here as well, wouldn't we?

Reply:
Ah, yeah that’s right. Good catch.